.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: php
+
+dist: trusty
+sudo: false
+
+cache:
+  directories:
+    - $HOME/.composer/cache/files
+
+matrix:
+  include:
```

Comment:
We should directly test php 7.3 and php 7.4 as well

Reply:
Done https://github.com/netlogix/nlxProductComparison/pull/1/commits/c4aeb2bb669e0429e33fc5d13cc7af0f15954d88